### PR TITLE
Feature : Cart Management

### PR DESCRIPTION
**Pull request by yogesh-**
Cart Management Completed.

Preview Link - https://deploy-preview-5--sneakerzz.netlify.app/

https://user-images.githubusercontent.com/7193961/161433387-cf5b6e73-c2d6-47db-9779-2e1f01dfe181.mp4

### CODE REVIEW

File: src/components/footer/footer.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.bottom-static {
+  /* position: fixed; */
```
Comment (chanchal-panpaliya) [inline src/components/footer/footer.css:2]: remove unwanted code

File: src/pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.row {
+  display: grid;
+  grid-template-columns: 65% 35%;
+  padding-left: 20rem;
+  padding-right: 20rem;
+  /* background-color: var(--hover-btn-color); */
```
Comment (chanchal-panpaliya) [inline src/pages/cart/cart.css:6]: remove unwanted code

File: src/pages/cart/cart.jsx [hunk 3]
```diff
@@ -1,12 +1,154 @@
...
 export const Cart = () => {
-  return (
-    <div>
-      <Navbar />
-      <h1>This is some random test</h1>
-      <Footer />
-    </div>
-  );
+  const { cartState, cartDispatch } = useCart();
+  console.log(cartState, "from cart state");
```
Comment (chanchal-panpaliya) [inline src/pages/cart/cart.jsx:8]: remove console

**Review by chanchal-panpaliya — COMMENTED** [review]
need to be - refactor code , remove comments and console.

File: src/pages/cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  background-color: var(--hover-btn-color);
+}
+
+.empty {
+  height: 50rem;
+  background-color: var(--hover-btn-color);
+}
+
+.scroll-cart {
+  /* height: 40rem; */
```
Comment (tushargoyal812) [inline src/pages/cart/cart.css:20]: remove comments

File: src/pages/cart/cart.jsx [hunk 5]
```diff
@@ -1,12 +1,154 @@
...
+  const cartData = cartState.cartProducts;
+  console.log(cartState.cartProducts, "cartstate+cartproducts");
+  console.log(cartData, "from cart data");
```
Comment (tushargoyal812) [inline src/pages/cart/cart.jsx:11]: remove console

File: src/pages/products/products.jsx [hunk 6]
```diff
@@ -43,7 +44,7 @@ export const Products = () => {
 
   const getFinalItems = sortItems(getRatedItems, state.sortBy);
 
-  console.log("Final filter items", getFinalItems, load);
+  console.log("Final filter items", getFinalItems, load, cartState);
```
Comment (tushargoyal812) [inline src/pages/products/products.jsx:47]: remove console

**Pull request merged**